fields/field.textbox.php

```diff
@@ -310,6 +311,29 @@ public function displaySettingsPanel(XMLElement &$wrapper, $errors = null) {
 			$label->setAttribute('class', 'column');
 			$columns->appendChild($label);
 
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'no', 'hidden'
+			);
+			$columns->appendChild($input);
+
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'yes', 'checkbox'
+			);
+
+			if ($this->get('handle_unique') == 'yes') {
+				$input->setAttribute('checked', 'checked');
+			}
+
+			$label = Widget::Label(
+				__('%s Handles are unique', array(
+					$input->generate()
+				))
+			);
+			$label->setAttribute('class', 'column');
+			$columns->appendChild($label);
+
```

Comment:
No idea, I've worked this the same way; I don't recall using this methodology on my own extensions, my guess is that this was some compatibility issue where some browsers might not have sent the 'checkbox' data if unchecked and a value which is set to `yes` was staying as such.